galleries/examples/lines_bars_and_markers/multivariate_marker_plot.py

```diff
@@ -39,7 +39,7 @@
     t = Affine2D().scale(skill).rotate_deg(takeoff)
     m = MarkerStyle(SUCCESS_SYMBOLS[mood], transform=t)
     ax.plot(pos[0], pos[1], marker=m, color=cmap(thrust))
-fig.colorbar(plt.cm.ScalarMappable(norm=Normalize(0, 1), cmap=cmap),
+fig.colorbar(plt.cm.ScalarMappable(norm=Normalize(0, 1), cmap="plasma"),
```

Comment:
Now the `cmap` variable in unused here; so either remove it, or change it instead.

Reply:
If I change `cmap="plasma"` in the above line, I am getting this error, 
```
 ax.plot(pos[0], pos[1], marker=m, color=cmap(thrust))
                                            ^^^^^^^^^^^^
TypeError: 'str' object is not callable
```

Reply:
Works here. Are you sure you've edited the correct line (l.42)? It's strange that an edit here should cause a failure in l.41?!?

If the issue is still present please post the full traceback.